### ISSUE TITLE
Reduce LoC

### ISSUE THREAD

**Issue by bwoebi** [binarygenius/phpthewrongway#8]
> Understand this clearly: The ideal number of lines of code in any project is as few as possible! 

Yes, less lines of code are usually better… if you start from an over-engineered system.

But in general, not true. You are rendering code illegible by that. Collapsing every if statement into ternaries doesn't help.

I'd rephrase it more as "less instructions", "reduce to smallest possible set of classes and interfaces" or such. LoC is a bad measurement here.

**Comment by nicholasruunu** [comment]
Yeah, or it could be something like "The ideal number of lines of code in any project is as few as possible whilst being as clear and readable as possible."

**Comment by unixdigest** [comment]
Thanks guys, great feedback, I have changed it.
